flags.md

```diff
@@ -0,0 +1,141 @@
+```nim
+
+## From build.nims
+
+# Arch:
+
+--cpu:arm                             # tags: arm32
+--cpu:arm64                           # tags: arm64
+--cpu:i386                            # tags: x86
+
+--define:bit32                        # tags: arm32
+--define:bit32                        # tags: x86
+
+--gcc.path:/usr/bin                   # tags: arm64
+--gcc.exe:aarch64-linux-gnu-gcc       # tags: arm64
+--gcc.linkerexe:aarch64-linux-gnu-gcc # tags: arm64
+
+--passC:'-m32'                        # tags: x86, gcc
+--passL:'-m32'                        # tags: x86, gcc
```

Comment:
`-m32` compiles in 32-bit mode.

Source: [GCC - x86 Options > `-m`][gcc]

[gcc]: https://gcc.gnu.org/onlinedocs/gcc/x86-Options.html#index-m32-2